nemo_skills/pipeline/generate.py

```diff
@@ -297,13 +324,26 @@ def generate(
                     )
                     prev_tasks = None
 
+                    single_output_dir = f"{output_dir}{'/generation' if generation_type == GenerationType.generate else ''}"
```

Comment:
can we move all this logic to be inside get_cmd?

Reply:
added in 04c13d7